api/MODULE.bazel

```diff
@@ -0,0 +1,68 @@
+module(
+    name = "envoy_api",
+    version = "0",
+)
+
+bazel_dep(name = "abseil-cpp", version = "20240116.2", repo_name = "com_google_absl")
+bazel_dep(name = "bazel_skylib", version = "1.6.1")
+bazel_dep(name = "gazelle", version = "0.36.0", repo_name = "bazel_gazelle")
+bazel_dep(name = "googleapis", version = "0.0.0-20240326-1c8d509c5", repo_name = "com_google_googleapis")
+bazel_dep(name = "googletest", version = "1.14.0.bcr.1", repo_name = "com_google_googletest")
+bazel_dep(name = "grpc", version = "1.62.1", repo_name = "com_github_grpc_grpc")
+bazel_dep(name = "opencensus-proto", version = "0.4.1.bcr.1", repo_name = "opencensus_proto")
+bazel_dep(name = "protobuf", version = "27.0-rc2", repo_name = "com_google_protobuf")
+bazel_dep(name = "protoc-gen-validate", version = "1.0.4", repo_name = "com_envoyproxy_protoc_gen_validate")
+bazel_dep(name = "re2", version = "2024-05-01", repo_name = "com_googlesource_code_re2")
+bazel_dep(name = "rules_go", version = "0.48.0", repo_name = "io_bazel_rules_go")
+bazel_dep(name = "rules_proto", version = "6.0.0.bcr.1")
+bazel_dep(name = "xds", version = "0.0.0-20240423-555b57e", repo_name = "com_github_cncf_xds")
+
+switched_rules = use_extension("@com_google_googleapis//:extensions.bzl", "switched_rules")
+switched_rules.use_languages(
+    cc = True,
+    go = True,
+)
+
+non_module_deps = use_extension("//bazel:repositories.bzl", "non_module_deps")
+use_repo(
+    non_module_deps,
+    "com_github_bufbuild_buf",
+    "envoy_toolshed",
+    "opentelemetry_proto",  # TODO: Move to bzlmod
```

Comment:
This is now part of BCR: https://github.com/bazelbuild/bazel-central-registry/tree/main/modules/opentelemetry-proto